scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -815,35 +783,30 @@ def test_as_euler_degenerate_compare_algorithms():
 
     idx = angles[:, 1] == 0  # find problematic angles indices
 
+    # Rotation of the form A/B/A are rotation around symmetric
+    # Axes
+    seq = "".join([seq_tuple[0], seq_tuple[1], seq_tuple[0]])
+    if intrinsic:
+        # Extrinsinc rotation (wrt to global world at lower case
+        # Intrinsic (WRT the object itself) lower case.
```

Comment:
```suggestion
        # Extrinsinc rotation (wrt to global world at lower case
        # Intrinsic (WRT the object itself) lower case.
```

This has an unclosed bracket and describes both cases as lower-case, which isn't true. The same string appears at least 3 times, with the same issue.

Reply:
Both should be fixed.